pkg/porter/lifecycle.go

```diff
@@ -5,11 +5,22 @@ import (
 	"github.com/pkg/errors"
 )
 
+var _ BundleAction = BundleLifecycleOpts{}
+
+// BundleAction is an interface that defines methods related to constructing
+// ActionArguments and supplying BundleLifecycleOptions.  The latter is useful
+// when the implementation contains further, action-specific options beyond
+// the stock BundleLifecycleOptions.
+type BundleAction interface {
+	ToActionArgs(*dependencyExecutioner) cnabprovider.ActionArguments
```

Comment:
Looks like this isn't used anywhere? I thought it may be used depending on how the implementation worked out but guess not.

Reply:
it is used here: https://github.com/deislabs/porter/blob/4b2b64746636e6997ee1f3dae8c87953c8b0cddc/pkg/porter/dependencies.go#L86

But, to your point, it could just as easily be removed from this interface and exist as it once was (a method on `BundleLifecycleOpts`), since at this time we don't see a need to have BundleAction implementations overriding its behavior.  Shall I change this back and remove the method from the interface?

Reply:
Oh, thanks I obviously missed that. Either way is fine, up to you!

Reply:
Ok, removed from interface 👍 .